build.sbt

```diff
@@ -52,6 +52,7 @@ val dateFormatter = java.time.format.DateTimeFormatter.BASIC_ISO_DATE
 
 inThisBuild(
   List(
+    scalaVersion in ThisBuild := "2.13.3",
```

Comment:
Isn't this set by `sbt-gsp`?

Reply:
Yes for the most part but not the root one and `metals` was complaining. We have an older `sbt-gsp` here waiting for the scala.js upgrade